### PR TITLE
use ImageStreamImport for container command lookup in oc debug

### PR DESCRIPTION
**Pull request by jim-minter**
[test]

### CODE REVIEW

File: pkg/cmd/cli/cmd/debug.go [hunk 1]
```diff
@@ -394,18 +395,103 @@ func (o *DebugOptions) Debug() error {
...
+			if err != nil {
+				return nil, err
+			}
+
+			namespace := trigger.ImageChangeParams.From.Namespace
+			if namespace == "" {
+				namespace = o.Attach.Pod.Namespace
+			}
+
+			isi, err := o.Client.ImageStreamImages(namespace).Get(isname, ref.ID)
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/debug.go]: You need to check that ID is step.

File: pkg/cmd/cli/cmd/debug.go [hunk 2]
```diff
@@ -394,18 +395,103 @@ func (o *DebugOptions) Debug() error {
...
+			trigger.ImageChangeParams != nil &&
+			trigger.ImageChangeParams.From.Kind == "ImageStreamTag" {
+
+			isname, _, err := imageapi.ParseImageStreamTagName(trigger.ImageChangeParams.From.Name)
+			if err != nil {
+				return nil, err
+			}
+
+			namespace := trigger.ImageChangeParams.From.Namespace
+			if namespace == "" {
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/debug.go]: We use `len(namespace) == 0` by convention.

**Comment by jim-minter** [comment]
@smarterclayton requested changes made

**Comment by jim-minter** [comment]
@smarterclayton if this is good to go, please hit merge.

**Comment by jim-minter** [comment]
ping @smarterclayton ...

**Comment by smarterclayton** [comment]
The other issue we discussed has been opened, correct?  LGTM [merge]

**Pull request merged**